cmd/endpoint.go

```diff
@@ -182,12 +182,26 @@ func (endpoints EndpointList) SetHTTP() {
 func NewEndpointList(args ...string) (endpoints EndpointList, err error) {
 	// isValidDistribution - checks whether given count is a valid distribution for erasure coding.
 	isValidDistribution := func(count int) bool {
-		return (count >= 4 && count <= 16 && count%2 == 0)
+		return (count >= minErasureBlocks && count <= maxErasureBlocks && count%2 == 0)
+	}
+
+	// getValidErasureBlocks returns a string containing the valid values for erasure coding.
+	getValidErasureBlocks := func() string {
+		list := ""
+		for blocks := minErasureBlocks; blocks <= maxErasureBlocks; blocks += 2 {
+			list += fmt.Sprintf("%d", blocks)
+			if blocks < maxErasureBlocks-2 {
+				list += ", "
+			} else if blocks == maxErasureBlocks-2 {
+				list += " or "
+			}
+		}
+		return list
 	}
 
 	// Check whether no. of args are valid for XL distribution.
 	if !isValidDistribution(len(args)) {
-		return nil, fmt.Errorf("total endpoints %d found. For XL/Distribute, it should be 4, 6, 8, 10, 12, 14 or 16", len(args))
+		return nil, fmt.Errorf("total endpoints %d found. For XL/Distribute, it should be %s", len(args), getValidErasureBlocks())
```

Comment:
We can think about removing XL from logging .. to avoid confusion with our docs. We can just say `erasure` 

```A total of %d endpoints were found. For erasure mode it should be %d```